File: lenggiauit-portfolio/src/services/resources/blogPostResource.ts
import { CategoryResource } from "./categoryResource";
import { TagResource } from "./tagResource";
import { UserResource } from "./userResource";

 
export type BlogPostResource = {
    title: any,
    thumbnail: any,
    shortDescription: any,
    content: any,
    url: any,
    view: any,
    comment: any,
    user: UserResource,
    category: CategoryResource,
    tags: TagResource[],
    totalRows: any,
    createdDate: Date,
    updatedDate?: Date
}

export type BlogPostRelatedResource = {
    title: any, 
    url: any, 
}
 

File: lenggiauit-portfolio-api/Resources/BlogPostResource.cs
using System;
using System.Collections.Generic;

namespace Lenggiauit.API.Resources
{
    public class BlogPostResource
    {
        public string Title { get; set; }
        public string ShortDescription { get; set; }
        public string Thumbnail { get; set; }
        public string Content { get; set; }
        public string Url { get; set; }
        public int View { get; set; }
        public int Comment { get; set; } 
        public DateTime CreatedDate { get; set; }
        public DateTime UpdatedDate { get; set; }
        public int TotalRows { get; set; }
        public UserResource User { get; set; }
        public  CategoryResource Category { get; set; }
        public  List<TagResource> Tags { get; set; } 
    }
}
